GpxLocationManager/GpxLocationManager.swift

```diff
@@ -181,14 +181,14 @@ open class GpxLocationManager {
           if abs(timeBetweenExpectedUpdateAndNextLocation) < GpxLocationManager.dateFudge {
             if self.isUpdatingLocations {
               self.callerQueue.async(execute: {
-                self.delegate.locationManager?(self.dummyCLLocationManager, didUpdateLocations: [currentLocation])
+                self.delegate?.locationManager?(self.dummyCLLocationManager, didUpdateLocations: [currentLocation])
 
```

Comment:
While I'm thinking of it, please delete this line.

Reply:
These two lines are no longer necessary?